docs/latest/design/roadmap.md

```diff
@@ -32,13 +32,18 @@ contributing to the project.
 - Achieve Gateway API conformance (e.g. routing, LB, Header transformation, etc.)- [Issue #65][65].
 - Setup a CI/CD pipeline- [Issue #63][63].
 
-### [v0.3.0][v0.3.0]: Drive Advanced Features through Extension Mechanisms
+### [v0.3.0][v0.3.0]: Driving Advanced Features through Extension Mechanisms
```

Comment:
Related to @danehans's comment, maybe worth calling out the deliverable of establishing a pattern for all extensions, since this is a big part of what we're actually doing with the authn design.

Reply:
Good point, will add